cheetah/converters/elegant.py

```diff
@@ -309,6 +309,26 @@ def convert_element(
                 frequency=torch.tensor(parsed["frequency"], **factory_kwargs),
                 name=name,
             )
+        elif parsed["element_type"] == "kicker":
+            validate_understood_properties(
+                ["element_type", "hkick", "vkick"],
+                parsed
+            )
+            return cheetah.Segment(
+                elements=[
+                    cheetah.HorizontalCorrector(
+                        length=torch.tensor(parsed.get("l", 0.0)/2, **factory_kwargs),
```

Comment:
I arbitrarily put the Horizontal Corrector first, and gave it half the length. Not sure if this is the right approach.

Reply:
Maybe this could be done differently once #203 is merged.